server/test/controllers/applicant/ApplicantProgramsControllerTest.java

```diff
@@ -151,12 +151,11 @@ public void test_deduplicate_inProgressPrograms() {
             .join();
 
     assertThat(result.status()).isEqualTo(OK);
-    // A program's name appears in the index view page content 3 times:
+    // A program's name appears in the index view page content 2 times:
     //  1) Program card title
-    //  2) Program details aria-label
-    //  3) Apply button aria-label
+    //  2) Apply button aria-label
```

Comment:
suggestion (maybe blocking): Add an external url to this program so that we can still test that the program name shows up in the aria label (unless this is already tested elsewhere).

Reply:
Ohhh I see what you mean! Ok I think that aria-label thing is tested elsewhere so this I agree this is not needed.